connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -301,6 +301,15 @@ public void timestampToConnectOptional() {
         assertNull(schemaAndValue.value());
     }
 
+    @Test
+    public void timestampToConnectDefval() {
```

Comment:
Do you think it'd be worth adding a test similar to `logicalToConnectWithDefaultValue ` that also tests whether rehydrating a value given an _optional_ schema that has a default value?